akka-javasdk-testkit/src/main/scala/akka/javasdk/testkit/impl/TimedActionResultImpl.scala

```diff
@@ -16,7 +16,7 @@ final class TimedActionResultImpl[T](effect: TimedActionEffectImpl.PrimaryEffect
   def this(effect: TimedAction.Effect) = this(effect.asInstanceOf[TimedActionEffectImpl.PrimaryEffect])
 
   /** @return true if the call had an effect with a reply, false if not */
-  override def isDone(): Boolean = effect.isInstanceOf[TimedActionEffectImpl.ReplyEffect]
+  override def isDone(): Boolean = effect.isInstanceOf[TimedActionEffectImpl.SuccessEffect.type]
```

Comment:
singleton so `==` would be cleaner than an instanceOf

Reply:
fixed